scss/core/root.scss

```diff
@@ -44,17 +44,22 @@
   --fd-color-background-negative: #{map-get($fd-colors-background-states, "negative")};
 
   //padding and gutter values used in page, section, and container
-  --fd-padding-ui: #{map-get($fd-spacing, xxs)};
-  --fd-width-gutter: #{map-get($fd-spacing, xxs)};
-  @include fd-screen(s) {
-    --fd-padding-ui: #{map-get($fd-spacing, xs)};
-  }
+  --fd-padding-ui: #{map-get($fd-spacing, xxs)}; // 8px ~ Margin (A)
```

Comment:
@xak ~ Did I assume this right? What we call `--fd-padding-ui` is what they call `Margin (A)`?

Reply:
Yes!